packages/tools/devtools/devtools-core/src/messaging/container-devtools-messages/SendEditData.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { HasContainerKey, HasEditType, HasFluidObjectId, HasNewData } from "../../CommonInterfaces";
+import { IDevtoolsMessage } from "../Messages";
+
+/**
+ * Encapsulates types and logic related to {@link SendEditData.Message}.
+ *
+ * @internal
+ */
+export namespace SendEditData {
+	/**
+	 * {@link GetDataVisualization.Message} {@link IDevtoolsMessage."type"}.
+	 *
+	 * @internal
+	 */
+	export const MessageType = "SEND_EDIT_DATA";
+
+	/**
+	 * Message data format used by {@link GetDataVisualization.Message}.
+	 *
+	 * @internal
+	 */
+	export type MessageData = HasContainerKey & HasFluidObjectId & HasNewData & HasEditType;
+
+	/**
+	 * Inbound message requesting a visualization for a specific DDS via its associated {@link HasFluidObjectId.fluidObjectId}.
```

Comment:
Copy-paste. Let's be sure to document what this message represents here.

Reply:
Should be good now